plugins/search-backend-module-elasticsearch/api-report.md

```diff
@@ -211,10 +224,14 @@ export class ElasticSearchSearchEngine implements SearchEngine {
   }: ElasticSearchOptions): Promise<ElasticSearchSearchEngine>;
   // (undocumented)
   getIndexer(type: string): Promise<ElasticSearchSearchEngineIndexer>;
+  // (undocumented)
+  protected indexTemplate?: ElasticSearchCustomIndexTemplate;
   newClient<T>(create: (options: ElasticSearchClientOptions) => T): T;
   // (undocumented)
   query(query: SearchQuery): Promise<IndexableResultSet>;
   // (undocumented)
+  setIndexTemplate(template: ElasticSearchCustomIndexTemplate): void;
```

Comment:
Would be nice to document this.  I suppose [here](https://backstage.io/docs/features/search/search-engines#elasticsearch) is sort of the only appropriate place for it at this point?

Reply:
yes we will have a full how to guide for this coming, but for now I can add a piece in there!

Reply:
Added! https://github.com/backstage/backstage/pull/11962/commits/d3edcd078476e460db7a0e97e901d58c4170fde2